app/presenters/notices/setup/check-notice-type.presenter.js

```diff
@@ -18,56 +20,97 @@ const NOTICE_TYPE_TEXT = {
  * @returns {object} - The data formatted for the view template
  */
 function go(session) {
-  const { licenceRef, noticeType, id: sessionId } = session
+  const { dueReturns, id: sessionId, licenceRef, noticeType, selectedReturns = [] } = session
 
   return {
     continueButton: _continueButton(sessionId),
     pageTitle: 'Check the notice type',
-    summaryList: _summaryList(licenceRef, noticeType, sessionId)
+    summaryList: _summaryList(sessionId, licenceRef, noticeType, dueReturns, selectedReturns)
+  }
+}
+
+function _returns(sessionId, selectedReturns, dueReturns) {
+  const selectedDueReturns = dueReturns.filter((dueReturn) => {
+    return selectedReturns.includes(dueReturn.returnId)
+  })
+
+  const selectedDueReturnsNewLineString = selectedDueReturns
+    .map((returnItem) => {
+      return `${returnItem.returnReference} - ${formatLongDate(new Date(returnItem.startDate))} to ${formatLongDate(new Date(returnItem.endDate))}`
+    })
+    .join('<br>')
```

Comment:
I think I got told off for using a `<br>` for separating stuff!

But also this brings HTML layout into the code, when we should always be trying to go the other way.

So, we should just be returning the array of strings and letting the view handle the formatting. I'm pretty sure someone even [created a macro](https://github.com/DEFRA/water-abstraction-system/pull/1630) specifically for this. 🤔 😁